common/http.go

```diff
@@ -0,0 +1,131 @@
+package common
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"gopkg.in/go-playground/validator.v9"
+
+	"github.com/pkg/errors"
+)
+
+type ErrorResponse struct {
+	Success bool `json:"success,omitempty"`
```

Comment:
So ErrorResponse.Success is used in tendermint/basecoin/cmd/baseserver and tendermint/go-crypto/keys so IMO we still need it.